### PR TITLE
removed early-kernel-ship-ok tag

### PR DESCRIPTION
**Pull request by mbiarnes**
removed leftovers of: early-kernel-ship-ok tag

### CODE REVIEW

File: doozer/tests/cli/test_gen_payload.py [hunk 1]
```diff
@@ -727,7 +727,7 @@ def test_rpm_deliveries(self):
         )
         ai.check_installed_rpms_in_image.side_effect = lambda dg_key, bi: {
             "foo": [],
-            "bar": [Mock(AssemblyIssue, code=AssemblyIssueCode.MISSING_SHIP_OK_TAG, component="bar", msg="")]
+            "bar": [Mock(AssemblyIssue, code=AssemblyIssueCode.MISSING_RHCOS_CONTAINER, component="bar", msg="")]
```
Comment (thegreyd) [inline doozer/tests/cli/test_gen_payload.py:730]: Better to delete this test, since it doesn't make sense with this assemblyissue

Comment (mbiarnes) [reply doozer/tests/cli/test_gen_payload.py]: test removed

**Pull request merged**